dxtbx/model/__init__.py

```diff
@@ -542,6 +552,7 @@ def get_template(imset):
     result['scan']       = [s.to_dict() for s in slist if s is not None]
     result['crystal']    = [c.to_dict() for c in clist if c is not None]
     result['profile']    = [p.to_dict() for p in plist if p is not None]
+    result['scaling_model']    = [s.to_dict() for s in scalelist if s is not None]
```

Comment:
This indentation is probably my fault! Feel free to unalign the lines.

Reply:
Okay, I will unalign all relevant instances in the modified files.